dnn/data/augmentation/detection.py

```diff
@@ -2,6 +2,10 @@
 import tensorflow as tf
 import tensorflow_addons as tfa
 
+physical_devices = tf.config.experimental.list_physical_devices('GPU')
+for physical_device in physical_devices:
+    tf.config.experimental.set_memory_growth(physical_device, True)
+
```

Comment:
I don't think this should be called in this file global scope. Whoever responsible for the top-level script should call memory management stuff. And there's a function `dnn.utils.mem.tf_set_memory_growth` for this.